src/main/java/com/devcourse/checkmoi/domain/file/model/vo/AttachedFile.java

```diff
@@ -0,0 +1,98 @@
+package com.devcourse.checkmoi.domain.file.model.vo;
+
+import static org.apache.commons.io.FilenameUtils.getExtension;
+import com.devcourse.checkmoi.domain.file.exception.NotAllowedFileException;
+import java.io.IOException;
+import java.util.UUID;
+import org.springframework.util.ObjectUtils;
+import org.springframework.web.multipart.MultipartFile;
+
+
+public class AttachedFile {
+
+    private final String originalFileName;
+
+    private final String contentType;
+
+    private final byte[] bytes;
+
+    public AttachedFile(String originalFileName, String contentType, byte[] bytes) {
+        this.originalFileName = originalFileName;
+        this.contentType = contentType;
+        this.bytes = bytes;
+    }
+
+    private static boolean verify(MultipartFile multipartFile, FileType fileType) {
+        if (multipartFile != null && multipartFile.getSize() > 0
+            && multipartFile.getOriginalFilename() != null) {
+
+            String contentType = multipartFile.getContentType();
+
+            return !ObjectUtils.isEmpty(contentType) && contentType.toLowerCase()
+                .startsWith(fileType.getPrefix());
+        }
+        return false;
```

Comment:
&& 조건이 나열되어 있으면 저 조건을 집중해서 다 읽어야 하기도 해서
`실패하는 경우를 모두 || 로 명시`하면 어느 경우에 실패하는지 좀 더 직관적으로 나타낼 수 있다고 생각합니다, 또한 이렇게 작성되었을 경우 나중에 인덴트로 인해 가독성 줄어들 수 있다고 생각되어서

실패 조건과 함께 `가장 상단에서 false를 리턴`해주는 것이 좀 더 좋다고 생각되는데,
혹시 어떻게 생각하시나요?

```suggestion
        if (multipartFile == null || multipartFile.getSize() <= 0 || multipartFile.getOriginalFilename() == null) {
            return false;
        }
        
        String contentType = multipartFile.getContentType();
        return !ObjectUtils.isEmpty(contentType)
            && contentType.toLowerCase().startsWith(fileType.getPrefix());
```

Reply:
지금은 수빈님이 말씀하신 것 처럼 하는게 더 나은 것 같습니당! 수정하겠습니다! 굳!👍🏻